src/server/index.js

```diff
@@ -11,6 +11,8 @@ import { sessionCache } from '~/src/server/common/helpers/session-cache/session-
 import { getCacheEngine } from '~/src/server/common/helpers/session-cache/cache-engine.js'
 import { pulse } from '~/src/server/common/helpers/pulse.js'
 
+const enablePulse = config.get('enablePulse')
```

Comment:
maybe make it work like secure context. we moved the enabled check into the plugin itself rather than have it in createServer. makes it a bit cleaner imo?